service/prometheus/update.go

```diff
@@ -0,0 +1,41 @@
+package prometheus
+
+import (
+	"github.com/prometheus/prometheus/config"
+)
+
+// isManaged returns true if the given scrape config is managed by the prometheus-config-controller,
+// false otherwise.
+func isManaged(scrapeConfig config.ScrapeConfig) bool {
```

Comment:
I would like to put private methods under public ones.

Reply:
Done